### DISCUSSION TITLE
Stricter eslint rules

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#19] [Ideas]
I would prefer stricter rules. see https://eslint.vuejs.org/rules/

We could go with the `plugin:vue/recommended` preset
We could add some extra rules that I like, like `vue/order-in-components`

**Comment by AlZorba** [comment]
@francoiscrouzet isn't this apllied ?

**Reply by francoiscrouzet** [reply]
Yep, it's cf file `.eslintrc`
Open to changes when needed, add or remove rules, etc
